src/controllers/__tests__/products.test.js

```diff
@@ -119,4 +120,43 @@ describe('Products routes', () => {
       expect(res.body.data).toEqual(expect.objectContaining(mProduct));
     });
   });
+
+  describe('POST /:id/request', () => {
+    test('If product with passed id does not exist, return error with status code 400', async () => {
+      const mUser = {
+        firstName: 'Glenn',
+        lastName: 'Quagmire',
+        email: 'glennQQQ@email.com',
+        schoolName: 'Yale University',
+        password: 'gleN123',
+      };
```

Comment:
Instead of writing this code twice (same code in other test), you can add this code to the `describe('POST /:id/request', () => { ..... }` code block with `beforeEach()`.